src/entity/components/Interactable.ts

```diff
@@ -7,19 +7,25 @@ export type Interactable = ECSComponent<
   InteractableBrand,
   {
     text: string;
+    pickupRadius: number;
```

Comment:
It should probably be interaction radius instead. We may want to extend this in the future with a priority. If multiple things are in range.